### PR TITLE
pgweb: update to 0.11.12.

### PR DESCRIPTION
**Pull request by arete**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/pgweb/template [hunk 1]
```diff
@@ -9,7 +9,7 @@ maintainer="Gerardo Di Iorio <arete74@gmail.com>"
 license="MIT"
 homepage="https://sosedoff.github.io/pgweb"
```
Comment (classabbyamp) [inline srcpkgs/pgweb/template:10]:
```suggestion
homepage="https://sosedoff.github.io/pgweb"
changelog="https://github.com/sosedoff/pgweb/raw/master/CHANGELOG.md"
```

Comment (arete) [reply srcpkgs/pgweb/template:10]: Thanks for suggest, i have update the pull request.

Comment (classabbyamp) [reply srcpkgs/pgweb/template:10]: I don't see it. did you push it?

Comment (arete) [reply srcpkgs/pgweb/template:10]: ipdate

**Pull request merged**